src/Compiler/SyntaxTree/SyntaxTrivia.fsi

```diff
@@ -446,3 +456,13 @@ type SynBindingReturnInfoTrivia =
         /// The syntax range of the `:` token
         ColonRange: range option
     }
+
+/// Represents additional information for SynMemberSig.Member
+[<NoEquality; NoComparison>]
+type SynMemberSigMemberTrivia =
+    {
+        /// The syntax range of 'get, set'
+        GetSetKeyword: range option
```

Comment:
In general, don't you want get and set separately because of the uncertain amount of whitespace between them and around the comma?

https://sharplab.io/#v2:EYLgxg9gTgpgtADwGwBYA0AbEAzAzgHwFsIATAVwxgAIAZAQwzoFgAoAFwE8AHagWQ4Aq3GAAo6ASioBeVlUIxCwGFCoA3BlQAa0qgCI6JMLqoB3AJZsAFlQDmMNlUdPnTtC/ePc9oA=

Reply:
Yes, but I don't need this information in my use case.
I add trivia on a need-to-have basis to keep the peace with the performance overhead (however little in this case).